package.json

```diff
@@ -50,7 +47,7 @@
     "d3-force": "^2.1.1",
     "d3-quadtree": "^1.0.7",
     "d3-scale-chromatic": "^1.3.3",
-    "deck.gl": "8.4.0-alpha.4",
+    "deck.gl": "^8.4.0",
```

Comment:
I have the same question as @manzt. It seems like luma.gl deprecated the main `luma.gl` package in favor of the modular `@luma.gl/x` packages but deck.gl docs still suggest the main `deck.gl` package? Is this change required to fix #872?

Reply:
Yes, I think this is easier and less error prone than maintaining a bunch of different deps (and indeed this may have contributed to #872).  There is equivalent for `@luma.gl/*` so we still have those deps.  @manzt This is why we need the linting rule, because those deps are not explicitly labeled here.

Reply:
I'm sorry, I'm such a dummy, `@luma.gl/core` is the equivalent for luma.  I don't know why I thought it wasn't.  I'll fix that now.